policyengine_us/variables/gov/irs/credits/earned_income/eitc_eligible.py

```diff
@@ -17,18 +17,23 @@ def formula(tax_unit, period, parameters):
         meets_age_requirements = (age >= min_age) & (
             age <= eitc.eligibility.age.max
         )
-        inv_income_sources = eitc.eligibility.counted_inv_income
-        investment_income = (
-            add(tax_unit, period, inv_income_sources)
-            + max_(0, tax_unit("c01000", period))
-            + max_(
-                0,
-                tax_unit("tax_unit_rental_income", period)
-                - tax_unit("tax_unit_partnership_s_corp_income", period),
-            )
+        no_loss_capital_gains = max_(
+            0,
+            add(
```

Comment:
```suggestion
            tax_unit("capital_gains", period)
```

Reply:
Done.

Reply:
When I implement your suggestion, I get 64 errors from "make test".  Here is a typical error:
```
policyengine_us/variables/gov/irs/credits/earned_income/eitc_eligible.py:22: in formula
    tax_unit("capital_gains", period),
//Users/mrh/anaconda3/envs/policyengine-us/lib/python3.9/site-packages/policyengine_core/populations/population.py:100: in __call__
    self.entity.check_variable_defined_for_entity(variable_name)
//Users/mrh/anaconda3/envs/policyengine-us/lib/python3.9/site-packages/policyengine_core/entities/entity.py:52: in check_variable_defined_for_entity
    raise ValueError(message)
E   ValueError: You tried to compute the variable 'capital_gains' for the entity 'tax_units';
E   however the variable 'capital_gains' is defined for 'people'.
```
I'm going to skip this suggested code change.